pontoon/administration/templates/admin_project.html

```diff
@@ -287,8 +287,10 @@ <h3>
 
 {% block extend_css %}
   {% stylesheet 'admin_project' %}
+  {{ render_bundle('tag_admin', 'css') }}
```

Comment:
There's nothing in there, or is it?

Reply:
hmm, maybe not - i think i have another npm package installed locally that fills it - ill check...

Reply:
Where is `render_bundle` coming from?

Reply:
here https://github.com/ezhome/django-webpack-loader

Reply:
ill remove this as its not necessary for this PR